src/auth.config.ts

```
@@ -0,0 +1,16 @@
+import type { NextAuthConfig } from "next-auth"
+import GitHub from "next-auth/providers/github"
+import Google from "next-auth/providers/google"
+
+export default {
+  pages: { signIn: "/login" },
+  session: { strategy: "jwt" },
+  callbacks: {
+    async authorized({ auth, request: { nextUrl } }) {
+      if (nextUrl.pathname.startsWith("/home")) return !!auth
+      else if (!!auth) return Response.redirect(new URL("/home", nextUrl))
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)